### PR TITLE
[DB/Quest] Gaining Even More Acceptance

### PR DESCRIPTION
**Pull request by xDevICCI**
- QUEST: Gaining Even More Acceptance

##### CHANGES PROPOSED:

-  make quest repeteable !
-  add 25 reputation in database

###### ISSUES ADDRESSED:

Closes 

##### TESTS PERFORMED:

- in my server , one player reported this  !

https://www.youtube.com/watch?v=xn_MmJC5My4 video guide !

##### HOW TO TEST THE CHANGES:

- Go to NPC : Master Smith Burninate <The Thorium Brotherhood>
- Take Quest and complete material and see not repeteable

- In column no exist value 25 reputation

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/db_world/2019_04_26.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* QUEST :
```
Comment (poszer) [inline data/sql/updates/db_world/2019_04_26.sql]: Could you please remove these unnecessary lines? You can include them in the commit description.

File: data/sql/updates/db_world/2019_04_26.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ https://wowgaming.altervista.org/aowow/?quest=13662
+
+give 25 reputation then complete quest
```
Comment (poszer) [inline data/sql/updates/db_world/2019_04_26.sql]: unnecessary

File: data/sql/updates/db_world/2019_04_26.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+quest is Repeatable
```
Comment (poszer) [inline data/sql/updates/db_world/2019_04_26.sql]: unnecessary

File: data/sql/updates/db_world/2019_04_26.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+ *************/
```
Comment (poszer) [inline data/sql/updates/db_world/2019_04_26.sql]: unnecessary

**Comment by xDevICCI** [comment]
change ready @poszer  sorry !

**Comment by poszer** [comment]
> change ready @poszer sorry !

That was fast ! :+1:  Thank you. Can you also do not remove the first line from the generated *sql file from script, and include it in your commit please : 

Example : 
```sql
INSERT INTO `version_db_world` (`sql_rev`) VALUES ('XXXXXXXXXXXX');

UPDATE quest_template SET RewardFactionOverride1 = 2500 WHERE ID = 13662;
UPDATE quest_template SET flags = 1 WHERE ID = 13662;
```

**Comment by xDevICCI** [comment]
I have no idea how it is done generate values version world :(

**Comment by poszer** [comment]
> I have no idea how it is done generate values version world :(

Write me on discord, i'll explain you there @xDevICCI

**Pull request closed without merging**